css/style.css

```
@@ -2,153 +2,220 @@
+}
+
+.sectionWrap {
+  padding: 60px 0 70px;
+}
+
+.mainHead {
+  font-size: 37px;
+  line-height: 60px;
+}
+
+.subHead {
+  font-size: 14px;
+}
+
+.headWrap {
+  text-align: center;
+}
+
+.inner {
+  max-width: 980px;
+  width: 96%;
+  margin: auto;
+}
+
+.boxes {
+  display: flex;
+}
+
+.contentsWrap {
+  margin-top: 40px;
+}
+
+.contentsWrap .box {
+  width: 310px;
+}
+
+.contentsWrap .box + .box {
+  margin-left: 25px;
 }
 
 /*============================================================================*/
 header {
-  background-color:#fff;
-  position:sticky;
-  top:0;
-  box-shadow:rgba(0, 0, 0, 0.3) 1px 1px 3px;
+  background-color: #fff;
+  position: sticky;
+  top: 0;
+  box-shadow: rgba(0, 0, 0, 0.3) 1px 1px 3px;
 }
 
 header .inner {
-  max-width:980px;
-  width:96%;
-  height:60px;
-  display:flex;
-  justify-content:space-between;
-  align-items:center;
-  margin:auto;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
 }
 
 header .logo a {
-  display:flex;
-  width:100px;
+  display: flex;
+  width: 100px;
 }
 
 header .list {
-  display:flex;
-  align-items:center;
+  display: flex;
+  align-items: center;
 }
 
 header .list .item + .item {
-  margin-left:20px;
+  margin-left: 20px;
 }
 
 header .btn {
-  width:132px;
-  height:30px;
+  width: 132px;
+  height: 30px;
 }
 
 header .toggle {
-  display:none;
+  display: none;
 }
 
 .firstView {
-  background-image:url(../img/first-view.jpg);
-  background-size:cover;
-  background-position:center;
-  color:#fff;
-  padding-top:60px;
-  height:600px;
+  background-image: url(../img/first-view.jpg);
+  background-size: cover;
+  background-position: center;
+  color: #fff;
+  padding-top: 60px;
+  height: 600px;
 }
 
 .firstView .inner {
-  max-width:662px;
-  width:96%;
-  margin:auto;
+  max-width: 662px;
 }
 
 .firstView .text {
-  margin-top:40px;
-  text-align:center;
+  margin-top: 40px;
+  text-align: center;
 }
 
 .firstView .conditions {
-  border-top:1px solid #fff;
-  margin:40px auto 0;
-  max-width:500px;
-  padding-top:40px;
-  text-align:center;
-  width:100%;
+  border-top: 1px solid #fff;
+  margin: 40px auto 0;
+  max-width: 500px;
+  padding-top: 40px;
+  text-align: center;
+  width: 100%;
 }
 
 .firstView .btnWrap {
-  margin-top:40px;
+  margin-top: 40px;
 }
 
 .firstView .btn {
-  width:252px;
-  height:35px;
+  width: 252px;
+  height: 35px;
+}
+
+.section1 {
+  background-color: #f2faff;
+}
+
+.section1 .image {
+  width: 198px;
+  margin: auto;
+}
+
+.section1 .subHeading {
+  font-size: 16px;
+  margin-top: 20px;
+  text-align: center;
+  line-height: 1;
+}
+
+.section1 .text {
+  margin-top: 30px;
 }
 
 @media only screen and (max-width:1024px) {
 
   header .nav {
-    display:none;
+    display: none;
   }
 
   header .toggle {
-    display:block;
+    display: block;
+  }
+
+}
+
+@media only screen and (max-width:767px) {
+  
+  .inner .contentsWrap .box {
+    width: 100%;
+    margin: auto;
   }
```

```suggestion
  .box {
    width: 100%;
    margin: auto;
  }
```
あと2個目以降のboxに`margin-top: 40px;`入ってないです。

---

> あと2個目以降のboxに`margin-top: 40px;`入ってないです。

修正しました。
844a3eb71359a57888175a1dfb29083e3d3d8ceb